apisix/plugins/jwt-auth.lua

```diff
@@ -42,12 +42,11 @@ local schema = {
 
 local consumer_schema = {
     type = "object",
-    additionalProperties = false,
+    -- can't use additionalProperties with dependencies
```

Comment:
Is it a bug of JSONSchema?

Reply:
When `additionalProperties  = false`, you can't use `dependencies` to add new field. This is not explicitly shown in the spec, but both Lua and Go implementation have this limitation.

Reply:
ok, got it. many thx for your explain